src/components/Engines/styles.ts

```diff
@@ -2,15 +2,15 @@ import styled from 'react-emotion';
 import ListItem from '@material-ui/core/ListItem';
 import Typography from '@material-ui/core/Typography';
 
-export const Heading = styled(Typography)`
-  && {
-    font-weight: 700;
-    text-transform: capitalize;
-  }
-`;
+export const Heading = styled(Typography)({
+  '&&': {
+    fontWeight: 700,
+    textTransform: 'capitalize',
+  },
+});
```

Comment:
There is definetely a repetition in the Heading component styles we can refactor later 😄